### DISCUSSION TITLE
i can't put on sale my NFT domain name

### DISCUSSION THREAD

**Discussion by Ravshanama** [aptos-labs/aptos-developer-discussions#48] [Questions] [answered]
Hello. i can't put on sale my domain name. What i need to do?

Move abort in 0x1::object: ENO_UNGATED_TRANSFERS(0x50003): The object does not have ungated transfers enabled
![Новый рисунок (1)1](https://github.com/aptos-labs/aptos-developer-discussions/assets/118884792/3a8848f1-728b-42fe-a99e-9f895ec1bf26)
![Новый рисунок (2)2](https://github.com/aptos-labs/aptos-developer-discussions/assets/118884792/62b4a426-ff55-47f4-9aaf-840e08026bad)

**Comment by 0x-j** [comment]
I don't think the token is under your account, if you look at [your account on explorer](https://explorer.aptoslabs.com/account/0x5b6735ead28556348b9607c7b003e3c77a1031a353ca9ee954b50af598dd2ba0/tokens?network=mainnet), you don't have the name under the token tab. This probably means you transfer the name to other contracts. So the name still points to you, but you are not the current owner of the name NFT (i.e. token on Aptos standard)

**Reply by gregnazario** [reply, marked as answer]
It looks like you tried to send the transaction twice, it is actually already listed.

https://explorer.aptoslabs.com/txn/489481007?network=mainnet

**Reply by Ravshanama** [reply]
Thanks
